packages/nimbus/src/components/text-input/text-input.mdx

```diff
@@ -13,106 +13,180 @@ tags:
+const App = () => <TextInput tone="critical" placeholder="MM/DD/YYYY"/>
 ```
 
+> [!CAUTION]\
+> **Don't**
+>
+> - This placeholder text looks like it has been prefilled for the user which is more likely to confuse the user. 
+
+```jsx-live
+const App = () => <TextInput tone="critical" placeholder="01/21/1970"/>
+```
+
+### Size
+
+Allow for enough width for the user to read and add in their information. Avoid truncation if possible especially if the user will be referring back or needing to make sure there isn’t inaccurate information.  
+
+> [!TIP]\
+> When to use
+> 
+> - There is enough space that the user can easily consume and correct this information if needed.
+
+```jsx-live
+const App = () => <TextInput placeholder="01/21/1970" value="805 St Cloud Road"/>
+```
+
+> [!CAUTION]\
+> **Don't**
+>
+> - The user’s information is too long for this field to be able to read the details completely.
+
+```jsx-live
+const App = () => <TextInput value="805 St Cloud Road, Bel Air, Los Angeles, CA" />
+```
+
+## Specs
+
 ## Props
 
 <PropTable id="TextInput" />
+
+### Accessibility
+
+The component uses onPress* props to handle all user interactions—mouse clicks, touch taps, and keyboard presses—consistently.  These props (onPress, onPressStart, onPressEnd, etc.) ensure a smooth and accessible experience regardless of the user's input method or device.
+
+```jsx-live
+const App = () => <TextInput placeholder="XX/XX/XXXX" />
+```
+
+### Accessibility Standards
+
+**1. Perceivable:**
+  
+* **1.3.1 Info and Relationships:** Use proper `HTML (<label>, aria-labelledby, etc.)` to connect labels with their inputs. This helps assistive technologies (like screen readers) understand the connection and convey the information correctly.
+* **1.4.3 Contrast (Minimum):**  Ensure sufficient color contrast between the input field, its label, and the surrounding background. This makes it easier to read for people with low vision. The minimum contrast ratio is 4.5:1 for normal text and 3:1 for large text. 
+
+**2. Operable:**
+
+* **2.1.1 Keyboard:** Text inputs must be fully operable using only a keyboard (no mouse required). This includes navigating to the input, interacting with it, and submitting the form.
+* **2.4.6 Headings and Labels:** Labels should be clear, concise, and descriptive, accurately reflecting the purpose of the input field.
+
+**3. Understandable:**
+
+* **3.3.2 Labels or Instructions:** Provide clear labels or instructions to help users understand what type of information is expected in the input field.
+
+**4. Robust:**
+
+* **4.1.2 Name, Role, Value:** Ensure the input field's name (label), role (type of input), and current value are programmatically available to assistive technologies.
+
+**Key Considerations:**
+
+- **Error Messages:** When input errors occur, provide clear and specific error messages that are easy to understand and locate. Use ARIA attributes (aria-invalid, aria-describedby) to associate error messages with the input field.
```

Comment:
```suggestion
- **Error Messages:** When input errors occur, provide clear and specific error messages that are easy to understand and locate. Use ARIA attributes (`aria-invalid`, `aria-describedby`) to associate error messages with the input field.
```

Reply:
All feedback updated here: https://github.com/commercetools/nimbus/pull/79/commits/501e268817de81dfec9eb0fdf43b07e37780e0a1
Thank you